understanding/consistent-identification.html

```diff
@@ -126,13 +126,10 @@ <h2>事例</h2>
                <li>
 
 
-                  <p><strong>Example 8: Failure primarily impacting assistive technology users</strong></p>
+                  <p><strong>事例 8: 主に支援技術の利用者に影響を与える失敗例</strong></p>
 
 
-                  <p>Two buttons with the same functionality visually have the same text, but have been
-                     given
-                     different <code>aria-label="..."</code> accessible names. For users of assistive technologies,
-                     these two buttons will be announced differently and inconsistently.</p>
+                  <p>同じ機能を持つ二つのボタンは、視覚的には同じテキストであるが、<code>aria-label="..."</code> により異なるアクセシブルな名前 (accessible name) が付けられている。支援技術の利用者の場合、これら二つのボタンは異なる方法で一貫性のない方法で発表される。</p>
```

Comment:
```suggestion
                  <p>同じ機能を持つ二つのボタンは、視覚的には同じテキストであるが、<code>aria-label="..."</code> により異なるアクセシブルな名前 (accessible name) が付けられている。支援技術の利用者の場合、これら二つのボタンは異なった一貫性のないものとして告げられる。</p>
```

Reply:
「be announced」の「発表される」という訳がしっくりしない気がしたので、ちょっと変えてみましたが、いかがでしょうか？

Reply:
告げる、というのも意味的には通るかと思いますが、個人的には発表と大差ないような…？

「通知される」はいかがでしょうか？
```suggestion
                  <p>同じ機能を持つ二つのボタンは、視覚的には同じテキストであるが、<code>aria-label="..."</code> により異なるアクセシブルな名前 (accessible name) が付けられている。支援技術の利用者の場合、これら二つのボタンは異なる方法で一貫性のない方法で通知される。</p>
```